llvm/test/CodeGen/AMDGPU/GlobalISel/inst-select-scalar-float-sop2.mir

```diff
@@ -14,8 +14,8 @@ body:             |
     ; GFX1150-NEXT: {{  $}}
     ; GFX1150-NEXT: [[COPY:%[0-9]+]]:sreg_32 = COPY $sgpr0
     ; GFX1150-NEXT: [[COPY1:%[0-9]+]]:sreg_32 = COPY $sgpr1
-    ; GFX1150-NEXT: %2:sreg_32 = nofpexcept S_ADD_F32 [[COPY]], [[COPY1]], implicit $mode
-    ; GFX1150-NEXT: $sgpr0 = COPY %2
+    ; GFX1150-NEXT: [[S_ADD_F32_:%[0-9]+]]:sreg_32 = nofpexcept S_ADD_F32 [[COPY]], [[COPY1]], implicit $mode
```

Comment:
You could avoid some churn here by first regenerating the checks in this file and committing it as obvious, and then rebasing the current patch on top of that. Same for any other files which have unrelated changes like this.

Reply:
I cleaned it up, and it looks much better now.